browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -153,13 +153,15 @@ void RewardsDOMHandler::GetAllBalanceReports() {
     for (auto const& report : reports) {
       const brave_rewards::BalanceReport oldReport = report.second;
       auto newReport = std::make_unique<base::DictionaryValue>();
-      newReport->SetDouble("opening", oldReport.opening_balance);
-      newReport->SetDouble("closing", oldReport.closing_balance);
-      newReport->SetDouble("grants", oldReport.grants);
-      newReport->SetDouble("ads", oldReport.earning_from_ads);
-      newReport->SetDouble("contribute", oldReport.auto_contribute);
-      newReport->SetDouble("donations", oldReport.recurring_donation);
-      newReport->SetDouble("oneTime", oldReport.one_time_donation);
+      newReport->SetString("opening", oldReport.opening_balance);
```

Comment:
This list was updated so that it matches names and fields from https://github.com/brave/brave-ui/blob/2fb8c297371056e898ae9dc2abb137fc88c50973/src/features/rewards/walletSummary/index.tsx#L27